dist/mapdc.js

```diff
@@ -84368,10 +84364,25 @@ function mapdTable(parent, chartGroup) {
         if (_isGroupedData) {
           _chart.onClick(d);
         } else if (col.expression in _filteredColumns) {
-          clearColFilter(col.expression);
+          delete _columnFilterMap[col.expression];
+          // this doesn't work. It ~never~ worked.
+          // const filterArray = cols.map(c => _columnFilterMap[c.expression])
+          var filterArray = [];
+          _chart.removeFilteredColumn(col.expression);
+          if (filterArray.some(function (f) {
+            return f !== undefined && f !== null;
+          })) {
+            _chart.onClick(filterArray); // will update global filter Clear icon
+          } else {
+            _chart.filterAll();
+          }
+          (0, _coreAsync.redrawAllAsync)(_chart.chartGroup());
```

Comment:
In an ideal world, we'd populate the filterArray with just the filters that still exist, but it doesn't work. I don't want to make it work right now.

So instead, we just scrap all filters if you try to clear any of them. This is the same as 5.2's behavior.